### PR TITLE
Misc minor updates

### PR DESCRIPTION
**Pull request by bbrto21**
* Call first OnClearCurrent when destroying TizenSurfaceGL
* Add LogLastEGLError from the Android Shell
* Move a method for resizing egl window to TizenNativeEGLWindow
* Tidy up

Signed-off-by: Boram Bae <boram21.bae@samsung.com>

### CODE REVIEW

File: shell/platform/tizen/tizen_native_window.cc [hunk 1]
```diff
@@ -45,32 +88,47 @@ TizenNativeEGLWindow::TizenNativeEGLWindow(
...
+    }
+    egl_display_ = EGL_NO_DISPLAY;
+  }
+
+  if (egl_window_ != nullptr) {
     ecore_wl2_egl_window_destroy(egl_window_);
     egl_window_ = nullptr;
   }
-  if (egl_display_ != EGL_NO_CONTEXT) {
-    eglTerminate(egl_display_);
```
Comment (bwikbs) [inline shell/platform/tizen/tizen_native_window.cc:72]: Don't we need this here?

**Review by bwikbs — APPROVED** [review]

File: shell/platform/tizen/tizen_native_window.cc (hunk 1 above)
Comment (bwikbs) [reply shell/platform/tizen/tizen_native_window.cc:72]: Oh.. never mind :smile:

File: shell/platform/tizen/channels/text_input_channel.cc [hunk 2]
```diff
@@ -113,7 +112,9 @@ void TextInputChannel::InputPanelStateChangedCallback(
               int32_t surface_w = window_geometry.w;
               int32_t surface_h =
                   window_geometry.h - self->current_keyboard_geometry_.h;
-              self->engine_->tizen_surface->SetSize(surface_w, surface_h);
+
+              self->engine_->tizen_native_window->GetTizenNativeEGLWindow()
+                  ->ReSizewidthRotation(0, 0, surface_w, surface_h, 0);
```
Comment (swift-kim) [inline shell/platform/tizen/channels/text_input_channel.cc:117]:
```suggestion
                  ->ResizeWithRotation(0, 0, surface_w, surface_h, 0);
```

File: shell/platform/tizen/external_texture_gl.h [hunk 3]
```diff
@@ -28,7 +28,7 @@ class ExternalTextureGL {
   /**
    * Returns the unique id for the ExternalTextureGL instance.
    */
-  int64_t texture_id() { return (int64_t)texture_id_; }
+  int64_t Texture_id() { return (int64_t)texture_id_; }
```
Comment (swift-kim) [inline shell/platform/tizen/external_texture_gl.h:31]:
```suggestion
  int64_t TextureId() { return (int64_t)texture_id_; }
```

File: shell/platform/tizen/tizen_native_window.cc [hunk 4]
```diff
@@ -6,6 +6,49 @@
 
 #include "flutter/shell/platform/tizen/tizen_log.h"
 
+void LogLastEGLError() {
```
Comment (swift-kim) [inline shell/platform/tizen/tizen_native_window.cc:9]: Should we add a copyright notice at the top when copying code from elsewhere?

**Review by swift-kim — APPROVED** [review]

File: shell/platform/tizen/tizen_native_window.cc (hunk 4 above)
Comment (bbrto21) [reply shell/platform/tizen/tizen_native_window.cc:9]: yes

File: shell/platform/tizen/channels/text_input_channel.cc (hunk 2 above)
Comment (bbrto21) [reply shell/platform/tizen/channels/text_input_channel.cc]: done

File: shell/platform/tizen/external_texture_gl.h (hunk 3 above)
Comment (bbrto21) [reply shell/platform/tizen/external_texture_gl.h]: done

**Pull request merged**